core/context.go

```diff
@@ -560,8 +566,13 @@ func NewContext(dbPath string, dbType string, dbName string, dbCount int) (*Cont
 		return nil, err
 	}
 
+	initCalcDebugConfig(ctx, debugConfigPath)
+
+	// Open calculation debug result DB
+	ctx.calcDebugDB = db.Open(ctx.DB.info.DBRoot, string(db.GoLevelDBBackend), "calculation_debug")
```

Comment:
Do not open initialize process.

Open at start of the calculation if calculation debugging is enabled.

Close at end of the calculation if calcDebugDB was opened.

Users want to query calculation debug result without icon_rc termination.

Reply:
Change DB open/close points for optimization. 
Open and close DB when write the calculation debug result.